packages/cli/src/lib/bundler/transforms.ts

```diff
@@ -90,15 +90,11 @@ export const transforms = (options: TransformOptions): Transforms => {
       },
     },
     {
-      test: /\.(eot|woff|woff2|ttf)$/,
-      use: [
-        {
-          loader: require.resolve('file-loader'),
-          options: {
-            name: 'static/[name].[hash:8].[ext]',
-          },
-        },
-      ],
+      test: /\.(eot|woff|woff2|ttf)$/i,
+      type: 'asset/resource',
+      generator: {
+        filename: 'static/[hash][ext][query]',
```

Comment:
I would like to keep the `[name]` prefix in there just to make it a bit easier to navigate the output and discover problems. Curious also about the query, do you know why it's in there? I see it's default

Reply:
I wondered that too, I think it is for the case where a loader might have a query string that you sometimes see.

Reply:
> Thank you! 👍
> 
> Should definitely switch to asset modules for all the assets. I'm curious, what is the bug that you encountered with the current setup?

You need an actual font-file which in my case was a .ttf looked like this

```ts
@font-face {
  font-family: 'GeosansLight';
  src: url('../fonts/GeosansLight/GeosansLight.ttf');
}
```

You get this error

> Should not import the named export 'version'.'split' (imported as 'version') from default-exporting module (only default export is available soon)

Not the most helpful error but changing the `file-loader` for the assets module fixed it.